src/cc/bcc_proc.h

```diff
@@ -34,6 +34,7 @@ int bcc_procutils_each_module(int pid, bcc_procutils_modulecb callback,
                               void *payload);
 int bcc_procutils_each_ksym(bcc_procutils_ksymcb callback, void *payload);
 void bcc_procutils_free(const char *ptr);
+char *bcc_procutils_language(int pid);
```

Comment:
We are returning a strdup'd string, so someone has to clean it up, no? Why not just return a const pointer to one of the static strings you have in the `languages` array?